common/fs/src/cache/mod.rs

```diff
@@ -1153,6 +1157,7 @@ impl FileSystem {
         };
 
         let path = entry.path().to_path_buf();
+        let _ = self.deletion_ack_sender.try_send(vec![path.clone()]);
```

Comment:
can it fail?

Reply:
Since the channel is unbounded, it will only fail if the receiver is dropped. The only time that might happen is if the agent is in process of exiting. I added a warn log to catch any errors just to be safe.